src/libNode/Node.cpp

```diff
@@ -384,8 +382,7 @@ bool Node::ValidateDB() {
       dirBlocks.emplace_back(*fallbackwshardingstruct);
     }
   }
-
-  if (!m_mediator.m_validator->CheckDirBlocks(dirBlocks, dsComm, 0, dsComm)) {
+  if (!m_mediator.m_validator->CheckDirBlocks(dirBlocks, dsComm, 1, dsComm)) {
```

Comment:
@KaustubhShamshery what does the change in index from 0 to 1 mean?

Reply:
The genesis blocks are synthesized at beginning of the binary, so no need to verify